packages/app/src/app/components/WorkspaceSetup/steps/ChangePlan.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Stack, Button, Text, Icon } from '@codesandbox/components';
+import * as dashboardUrls from '@codesandbox/common/lib/utils/url-generator/dashboard';
+
+import { useURLSearchParams } from 'app/hooks/useURLSearchParams';
+import { useActions, useAppState } from 'app/overmind';
+import { StepProps } from '../types';
+import { StepHeader } from '../StepHeader';
+import { AnimatedStep } from '../elements';
+
+export const ChangePlan: React.FC<StepProps> = ({
+  onPrevStep,
+  onDismiss,
+  currentStep,
+  numberOfSteps,
+}) => {
+  const actions = useActions();
+  const { checkout } = useAppState();
+  const { getQueryParam } = useURLSearchParams();
+  const urlWorkspaceId = getQueryParam('workspace');
+
+  const handleSubmit = async e => {
+    e.preventDefault();
+    const result = await actions.checkout.convertToUsageBilling({
+      workspaceId: urlWorkspaceId,
+    });
+    if (result.success) {
+      window.location.href = dashboardUrls.portalOverview(urlWorkspaceId);
+    } else {
+      actions.addNotification({
+        message: result.error,
+        type: 'error',
+        timeAlive: 10,
+      });
+    }
+  };
+
+  return (
+    <AnimatedStep>
+      <Stack direction="vertical" gap={12} as="form" onSubmit={handleSubmit}>
+        <StepHeader
+          onPrevStep={onPrevStep}
+          onDismiss={onDismiss}
+          currentStep={currentStep}
+          numberOfSteps={numberOfSteps}
+          title="Review plan"
+        />
+
+        <Text color="#e5e5e5">You are switching to the following plan:</Text>
+
+        <Stack justify="space-between" gap={4}>
+          <Stack direction="vertical">
+            <Text size={6} color="#e5e5e5">
+              Pro plan
+            </Text>
+            <Text>{checkout.totalCredits} VM credits</Text>
+            <Text>{checkout.totalSandboxes} Sandboxes</Text>
+          </Stack>
+          <Text size={6} color="#e5e5e5">
+            ${checkout.totalPrice} / month
+          </Text>
+        </Stack>
+
+        <Text>
+          Additional VM credits are available on-demand for $0.018/credit.
```

Comment:
Isn't this value available in a constant?

Reply:
Yes, could be a good idea as it's used in a few places